data/Music/Mario & Luigi Series/Mario & Luigi  Dream Team Bros/Never Let Up.meta

```diff
@@ -0,0 +1,3 @@
+Mario & Luigi: Dream Team Bros. - Never Let Up
```

Comment:
This should just be "Mario & Luigi: Dream Team". Could you also change the directory?
```suggestion
Mario & Luigi: Dream Team - Never Let Up
```